gsoc/filters.py

```diff
@@ -0,0 +1,40 @@
+from datetime import datetime
+
+from django.utils.translation import gettext_lazy as _
+from django.contrib.admin import SimpleListFilter
+
+
+class GSoCYearFilter(SimpleListFilter):
```

Comment:
This entire class isnt really necessary, no need to make a new file and class for one function, just stick this code in admin.py also doing it this way is like 2x the amount of code

Reply:
Fixed